bpaotu/bpaotu/frontend/package.json

```diff
@@ -74,5 +74,8 @@
     "webpack-bundle-analyzer": "^2.13.0",
     "webpack-cli": "^2.1.3",
     "webpack-merge": "^4.1.2"
+  },
```

Comment:
This should probably go into dev dependencies, just like all the others.

So remove with yarn and re-add with --dev.